worker/crates/action-universal/src/excel/writer.rs

```diff
@@ -0,0 +1,376 @@
+        let value = parts.next().ok_or_else(|| error::Error::internal_runtime("Invalid formatting value"))?;
+        match key {
+            "font" => format.set_font_name(value),
+            "size" => format.set_font_size(value.parse().map_err(|_| error::Error::internal_runtime("Invalid font size".to_string()))),
+            "color" => format.set_font_color(value),
+            "bold" => format.set_bold(),
+            "italic" => format.set_italic(),
+            "underline" => format.set_underline(str_to_format_underline(value)?),            
+            "background_color" => format.set_background_color(value),
+            "align" => format.set_align(str_to_format_align(value)?),
+            "wrap" => format.set_text_wrap(),
+            _ => return Err(error::Error::internal_runtime("Unknown formatting key")),
+        };
+    }
+    Ok(format)
+}
+
+// Row Format works with worksheet directly
+// fn parse_row_formatting(row_formatting: &str, worksheet: &worksheet) -> Result<Format> {
+//     let mut format = Format::new();
+//     for pair in row_formatting.split(';') {
+//         let mut parts = pair.splitn(2, ',');
+//         let key = parts.next().ok_or_else(|| error::Error::internal_runtime("Invalid row formatting key"))?;
+//         let value = parts.next().ok_or_else(|| error::Error::internal_runtime("Invalid row formatting value"))?;
+//         match key {
+//             "row_height" => format.set_row_height(value.parse().map_err(|_| error::Error::internal_runtime("Invalid row height"))?),
+//             _ => return Err(error::Error::internal_runtime("Unknown row formatting key")),
+//         }
+//     }
+//     Ok(format)
+// }
+
+fn str_to_format_underline(s: &str) -> Result<FormatUnderline, error::Error> {
+    match s {
+        "None" => Ok(FormatUnderline::None),
+        "Single" => Ok(FormatUnderline::Single),
+        "Double" => Ok(FormatUnderline::Double),
+        "SingleAccounting" => Ok(FormatUnderline::SingleAccounting),
+        "DoubleAccounting" => Ok(FormatUnderline::DoubleAccounting),
+        _ => Err(error::Error::internal_runtime("Invalid underline value")),
+    }
+}
+
+fn str_to_format_align(s: &str) -> Result<FormatAlign, error::Error> {
+    match s {
+        "General" => Ok(FormatAlign::General),
+        "Left" => Ok(FormatAlign::Left),
+        "Center" => Ok(FormatAlign::Center),
+        "Right" => Ok(FormatAlign::Right),
+        "Fill" => Ok(FormatAlign::Fill),
+        "Justify" => Ok(FormatAlign::Justify),
+        "CenterAcross" => Ok(FormatAlign::CenterAcross),
+        "Distributed" => Ok(FormatAlign::Distributed),
+        "Top" => Ok(FormatAlign::Top),
+        "Bottom" => Ok(FormatAlign::Bottom),
+        "VerticalCenter" => Ok(FormatAlign::VerticalCenter),
+        "VerticalJustify" => Ok(FormatAlign::VerticalJustify),
+        "VerticalDistributed" => Ok(FormatAlign::VerticalDistributed),
+        _ => Err(error::Error::internal_runtime("Invalid alignment value")),
+    }
+}
```

Comment:
TryFrom can be used
```rust
impl TryFrom<&str> for FormatAlign {
    type Error = /* */;

    fn try_from(value: i32) -> Result<Self, Self::Error> {
		// ...
	}
}
```

Reply:
I thought of doing that but `FormatUnderline` or `FormatAlign` is not defined in the current crate but part of https://docs.rs/rust_xlsxwriter/0.64.1/rust_xlsxwriter/index.html

Reply:
@pyshx 
I think we can implement TryFrom to FormatAlign for this.
Within this writer.rs file. The best thing to do would be to create a wrapper for Excel in common crate and implement TryFrom there as well, but I think we're good there.

Reply:
Understood! Can you check this https://github.com/reearth/reearth-flow/pull/138/commits/70fa5eafbe4b42048480414535b0ec200c1ec3a7 🙏🏼

Reply:
@pyshx 
Sorry, I had to comment. 🙇